helsingborg-stad/helsingborg-io-sls-api
# refactor(getattachmentlist): remove unused function

## Explain the changes you’ve made
Remove unused function

## Explain why these changes are made
- Removed unused function
- Remove policy needed for running the function
- Removed function tests

## How to test

1. Checkout this branch
2. run `sls deploy` within service folder for `service users-api`
3. Run a form of your liking and check that attachments get uploaded correctly
